ChangeLog.rst

```diff
@@ -42,6 +42,8 @@ Kubespray
 
 :ghpull:`223` - pre-provision Kibana index configuration (:ghissue:`174`)
 
+:ghpull:`274` - Add support for python3.7
```

Comment:
Nit-pick, but: for consistency, no capital here.

Reply:
Oups and added in wrong section